config.yml

```diff
@@ -48,6 +43,9 @@ defaults:
       secretAccessKey:  !env AWS_SECRET_ACCESS_KEY
       region:           us-west-2
       apiVersion:       2014-01-01
+  monitor:
+    mock:             true
```

Comment:
in other `main.js` files, we usually compare the environment to production and use that truthiness value instead of coding a `true` or `false` into the config files.  not too important.